app/src/main/java/com/github/se/polyfit/ui/screen/OverviewScreen.kt

```diff
@@ -0,0 +1,343 @@
+        border = BorderStroke(2.dp, MaterialTheme.colorScheme.primaryContainer),
+        elevation = ButtonDefaults.buttonElevation(defaultElevation = 4.dp),
+        colors =
+            ButtonDefaults.buttonColors(
+                contentColor = MaterialTheme.colorScheme.primary,
+                containerColor = MaterialTheme.colorScheme.primaryContainer),
+        modifier =
+            Modifier.align(Alignment.BottomStart)
+                .padding(start = 130.dp, bottom = 15.dp)
+                .background(Color.Transparent)) {
+          Icon(
+              painter = painterResource(R.drawable.baseline_mode_edit_outline_24),
+              contentDescription = null)
+        }
+
+    Button(
+        onClick = { /*TODO*/},
+        elevation = ButtonDefaults.buttonElevation(defaultElevation = 4.dp),
+        border = BorderStroke(2.dp, MaterialTheme.colorScheme.primaryContainer),
+        colors =
+            ButtonDefaults.buttonColors(
+                contentColor = MaterialTheme.colorScheme.primary,
+                containerColor = MaterialTheme.colorScheme.primaryContainer),
+        modifier =
+            Modifier.align(Alignment.BottomStart)
+                .padding(start = 235.dp, bottom = 15.dp)
+                .background(Color.Transparent)) {
+          Icon(
+              painter = painterResource(R.drawable.outline_assignment_24),
+              contentDescription = null)
+        }
+  }
+}
+
+private fun callCamera(
+    context: Context,
+    startCamera: ManagedActivityResultLauncher<Intent, ActivityResult>,
+    requestPermissionLauncher: ManagedActivityResultLauncher<String, Boolean>
+): () -> Unit = {
+  // Check if the permission has already been granted
+  when (ContextCompat.checkSelfPermission(context, Manifest.permission.CAMERA)) {
+    PackageManager.PERMISSION_GRANTED -> {
+      // You can use the camera
+      val takePictureIntent = Intent(MediaStore.ACTION_IMAGE_CAPTURE)
+      try {
+        startCamera.launch(takePictureIntent)
+      } catch (e: Exception) {
+        // Handle the exception if the camera intent cannot be launched
+        Log.e("HomeScreen", "Error launching camera intent: $e")
+      }
+    }
+    else -> {
+      // Request the permission
+      requestPermissionLauncher.launch(Manifest.permission.CAMERA)
+    }
+  }
+}
+
+@Composable
+fun OverviewContent(paddingValues: PaddingValues) {
+
+  val context = LocalContext.current
+
+  // State to hold the URI, the image and the bitmap
+  var imageUri by remember { mutableStateOf<Uri?>(null) }
+  val iconExample = BitmapFactory.decodeResource(context.resources, R.drawable.picture_example)
+  var imageBitmap by remember { mutableStateOf<Bitmap?>(iconExample) }
+
+  // Launcher for starting the camera activity
+  val startCamera =
+      rememberLauncherForActivityResult(ActivityResultContracts.StartActivityForResult()) { result
+        ->
+        val bitmap = result.data?.extras?.get("data") as? Bitmap
```

Comment:
The **get** method is deprecated. It would be a good idea to use a non-deprecated and equivalent method if one exists.